src/components/SelectionList/CardListItem.tsx

```diff
@@ -65,7 +65,7 @@ function CardListItem<TItem extends ListItem>({
                     </View>
                 )}
                 <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch, styles.optionRow]}>
-                    <View style={[styles.flexRow, styles.alignItemsCenter]}>
+                    <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch, styles.optionRow]}>
```

Comment:
```suggestion
                    <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch]}>
```

Needs to be removed because it adds extra padding:
```js
optionRow: {
    minHeight: variables.optionRowHeight,
    paddingTop: 12,
    paddingBottom: 12,
}
```

which doubles the height of the CardListItem:

| ❌ With the style | ✅ Without the style |
| ------------- | ------------- |
| <img width="300" alt="Screenshot 2024-08-21 at 16 32 41" src="https://github.com/user-attachments/assets/611353c9-bf04-4187-b35c-d744258fe88b"> | <img width="300" alt="Screenshot 2024-08-21 at 16 32 06" src="https://github.com/user-attachments/assets/beb5f249-7b10-42d1-8c4d-4a685a01bddc"> |